resources/definitions/hellbot_hidra.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "Hellbot Hidra",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Hellbot Development Team",
+        "manufacturer": "Hellbot",
+        "file_formats": "text/x-gcode",
+		"platform": "hellbot_hidra.obj",
+		"platform_offset": [0, 0, 5],
+        "platform_texture": "hellbot_hidra.png",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "hellbot_hidra_extruder_0",
+			"1": "hellbot_hidra_extruder_1"
+        }
+
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Hellbot Hidra" },
+        "machine_width": {
+            "default_value": 220
+        },
+		"machine_depth": {
+            "default_value": 220
+        },  
+        "machine_height": {
+            "default_value": 250
+        },      
+		"machine_heated_bed": {
+			"default_value": true 
+		},
+        "machine_center_is_zero": {
+            "default_value": false
+        },
+		"machine_head_with_fans_polygon":
+        {
+            "default_value": [
+                [ -75, 35 ],
+                [ -75, -18 ],
+                [ 18, 35 ],
+                [ 18, -18 ]
+            ]
+        },
+        "machine_extruder_count": {
+            "default_value": 2
+        },
+		"machine_start_gcode": {
+             "default_value": "G21; Unidades en Milimetro\nG90; Posicionamiento Absoluto\nM82; E Absoluto\nM107; Apagar Venitilador de capas\nG28; Llevar ejes a origen\nG1 Z15.0 F9000; Levantar Eje Z 15mm"
+        },
+        "machine_end_gcode": {
+            "default_value": "M104 T0 S0; Apagar Extrusor E0\nM104 T1 S0; Apagar Extrusor E1\nM140 S0; Apagar Cama Caliente\nG92 E1; Posicionar Extrusor en 1mm\nG1 E-1 F300; Retraer Extrusor 1mm\nG28 X0 Y0; Llevar al origen ejes X e Y\nM84; Desactivar Motores "
```

Comment:
Cura is primarily English, including all of its g-code documentation. Could you translate this to English?